Makefile

```diff
@@ -200,27 +127,14 @@ clean_python_pb:
 cleanall : clean
 	-@rm -rf proto/*.pb.c proto/*.pb.h
 
-load: all
-	python -m ledgerblue.loadApp $(APP_LOAD_PARAMS)
-
-load-offline: all
-	python -m ledgerblue.loadApp $(APP_LOAD_PARAMS) --offline
-
-delete:
-	python -m ledgerblue.deleteApp $(COMMON_DELETE_PARAMS)
-
-# import generic rules from the sdk
-include $(BOLOS_SDK)/Makefile.rules
-
 #add dependency on custom makefile filename
 dep/%.d: %.c Makefile
```

Comment:
Should not be needed. Deps are already handled by SDK Makefiles :thinking:

Reply:
Will take a look and check if I can remove it entirely.

Reply:
Removed it.